x-pack/plugins/cases/server/saved_object_types/migrations/comments.ts

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
This functionality was in `index.ts` originally in 7.15. We extracted it out to a new file in 7.16.